### DISCUSSION TITLE
Will this replace fvwm2 in OpenBSD & How does this improve upon fvwm2?

### DISCUSSION THREAD

**Discussion by ghost** [fvwmorg/fvwm3#596] [General]
More interested in the second question

**Comment by somiaj** [comment]
The biggest change/improvement at this point is RandR support, which has allowed for lots of different things when working with multiple monitors. Beyond just overall better multiple monitor support there is also `DesktopConfiguration per-monitor` (or `shared`) which are two different ways to allow you to treat the Desks/Pages on each monitor independently.

There have been a lot of little improvements as well, from cleaning up the code, to small extensions, you can look at the changelog to see a list of the major things that have changed. The best way to see improvements is just try it out, it should work for the most part with your current fvwm2 configuration, or you can use the default-config as a starting point.

As for what OpenBSD will do, you will have to ask them. I assume they will still keep 2.2.5 as the default fvwm version, but if they replace fvwm2 with fvwm3 in their extended packages or have both available will be up to the OpenBSD maintainers.

**Comment by harridu** [comment]
fvwm3 is still a little bit weird wrt multi-monitor support. Icons jump from one monitor to another, monitors are "jumped over" if you move a window, others? On a single big screen it works very well, but for multi-head support you might want to consider to wait for the next version.
